syntax = "proto3";

option go_package = "pb/game";

message ConnetionParam
{
  string ip = 1;
  uint32 port = 2;
  string db_host = 3;
  string db_user = 4;
  string db_password = 5;
  uint32 db_port = 6;
  string db_dbname = 7;
}

message GameConfigInfo
{
  uint32 zone_id = 1;
  uint32 lobby_id = 2;
  uint32 loglevel = 3;
  uint32 server_type = 4;
  repeated string mq_topics = 5;
  uint32 message_body_size = 7;
}


